internal/clients/kibana/alerting.go

```diff
@@ -0,0 +1,181 @@
+package kibana
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/elastic/terraform-provider-elasticstack/generated/alerting"
+	"github.com/elastic/terraform-provider-elasticstack/internal/clients"
+	"github.com/elastic/terraform-provider-elasticstack/internal/models"
+	"github.com/elastic/terraform-provider-elasticstack/internal/utils"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+)
+
+func ruleResponseToModel(spaceID string, res *alerting.RuleResponseProperties) *models.AlertingRule {
```

Comment:
Does it make sense to add unit tests to cover converting from TF data to the internal model, from the model to the request data and the opposite flow? The only tests we have are the acceptance tests.